components/calendar/calendar.js

```diff
@@ -490,39 +486,36 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 
 		const dates = getDatesInMonthArray(this._shownMonth, this._shownYear);
 		const dayRows = dates.map((week) => {
-			const weekHtml = week.map((day) => {
+			const weekHtml = week.map((day, index) => {
 				const disabled = getDisabled(day, this.minValue, this.maxValue);
 				const focused = checkIfDatesEqual(day, this._focusDate);
 				const selected = this.selectedValue ? checkIfDatesEqual(day, getDateFromISODate(this.selectedValue)) : false;
 				const classes = {
 					'd2l-calendar-date': true,
+					'd2l-calendar-date-disabled': disabled,
 					'd2l-calendar-date-initial': this._isInitialFocusDate,
 					'd2l-calendar-date-selected': selected,
 					'd2l-calendar-date-today': checkIfDatesEqual(day, this._today)
 				};
 				const year = day.getFullYear();
 				const month = day.getMonth();
 				const date = day.getDate();
-				const description = `${date}. ${selected ? this.localize('selected') : this.localize('notSelected')} ${formatDate(day, {format: 'monthYear'})}`;
+				const weekday = calendarData.descriptor.calendar.days.long[calendarData.daysOfWeekIndex[index]];
+				const description = `${weekday} ${date}. ${selected ? this.localize('selected') : this.localize('notSelected')} ${formatDate(day, {format: 'monthYear'})}`;
 				// role="gridcell" used for screen reader (e.g., JAWS and VoiceOver) behavior to work properly
 				return html`
 					<td
-						aria-selected="${selected ? 'true' : 'false'}"
+						aria-disabled="${disabled}"
+						aria-selected="${selected}"
```

Comment:
I think this should still be a string and not a boolean being passed in... it'll probably `.toString()` it and still work but just for safety maybe?

Reply:
I think I actually ran into a problem with this (using a boolean) not too long ago - I think with tabs.  I don't recall whether it was a linting/analyzer error, or an actual error though.